src/main/java/io/vertx/core/eventbus/EventBus.java

```diff
@@ -261,16 +261,35 @@
    *
    * @param interceptor  the interceptor
    * @return a reference to this, so the API can be used fluently
+   * @deprecated use {@link EventBus#addInterceptor(Interceptor)} instead.
    */
+  @Deprecated
   EventBus addInterceptor(Handler<SendContext> interceptor);
```

Comment:
remove it instead of deprecating it, this API was never really advertised and supported. So I prefer to finish it, document it and support it, instead of deprecating things.